netpyne/batch/batch.py

```diff
@@ -637,8 +646,10 @@ def evaluator(candidates, args):
                             jobNamePath = genFolderPath + "/gen_" + str(ngen) + "_cand_" + str(candidate_index)
                             if os.path.isfile(jobNamePath+'.json'):
                                 with open('%s.json'% (jobNamePath)) as file:
-                                    simData = json.load(file)['simData']
-                                fitness[candidate_index] = fitnessFunc(simData, **fitnessFuncArgs)
+                                    data = json.load(file)
+                                    simData = data['simData']
+                                    netData = data['net']
+                                fitness[candidate_index] = fitnessFunc(simData, netData, jobNamePath, **fitnessFuncArgs)
```

Comment:
@theiera did you update the examples that use fitnessFunc?

Reply:
the new fitnessFunc args are a good idea but not backward compatible; i.e. requires changing previously implemented fitnessFunc() to use netData and jobNamePath (eg. in examples/evolCell and M1 and A1 models)